lib/presets.js

```diff
@@ -305,15 +305,15 @@ module.exports = {
      * });
      */
     cargo: (options, cwd, version, callback) => {
-      const cargoToml = path.join(cwd, 'Cargo.toml');
+      const cargoFiles = path.join(cwd, 'Cargo.*');
```

Comment:
Do we really want such a wide-reaching glob, that would also match e.g. `Cargo.readme`?
(I've never done any Rust stuff, so I dunno if that's a silly question or not :wink: )

Reply:
It is only those two that are used for Rust projects. If someone puts a Cargo.xyz in the root, there still should be something inside which matches the regex. But people are doing strange things by nature and there really exists a chance for this by the universal laws. Although this conversation probably diminished a big part of it :D